Orange/widgets/unsupervised/owkmeans.py

```diff
@@ -130,6 +132,7 @@ class Warning(widget.OWWidget.Warning):
         not_enough_data = widget.Msg(
             "Too few ({}) unique data instances for {} clusters"
         )
+        no_sparse_normalization = widget.Msg("Sparse datacannot be normalized")
```

Comment:
Sparse datacannot be normalized --> Sparse data cannot be normalized

Reply:
Oops. Fixed.